spec/features/database_authenticatable_sign_in_feature_spec.rb

```diff
@@ -6,7 +6,7 @@
 
   context 'teacher signs in' do
     def expect_successful_sign_in_for(educator)
-      educator_sign_in(educator)
+      sign_in_attempt(educator.email, 'demo-password')
```

Comment:
I'm wondering how this works.  Won't this test fail since the password env variable isn't set?  Or wouldn't we want to test a sign in with a wrong password in this file?

Reply:
The config check sets 'demo-password' automatically in test and dev environments.

Reply:
Ah I see now, thanks!  Let's move this to `development.rb` and `test.rb`, then we can avoid any logic and code around setting env variables related to authorization automatically.  And in the production demo case, we have to set it manually in production for it to work.

`simple > easy`